internal/cmd/stack/open_command.go

```diff
@@ -149,10 +149,17 @@ func searchStacks(ctx context.Context, p *stackSearchParams) ([]stack, error) {
 	}
 
 	if p.projectRoot != nil {
+		roots := []graphql.String{graphql.String(*p.projectRoot)}
```

Comment:
@lexton Just to make sure this is resistant in face of changes in other parts of the repo, could you strip the "/" with strings.TrimSuffix on the project root here?

I know it doesn't have the trailing slash right now and will do nothing, but this wouldn't be fun to debug if it changed in the future.

Reply:
Yeah - totally agree - that's totally the right call :D